packages/core/src/babelPlugins/babel-plugin-redwood-routes-auto-loader.ts

```diff
@@ -1,8 +1,10 @@
 import type { PluginObj, types } from '@babel/core'
+import generate from '@babel/generator'
```

Comment:
debug code :)

Reply:
Yes, I wanted to get a run through of all the tests, with this in there. But yay! Everything passes ✔️  So will remove now